RUFAS/input_manager.py

```diff
@@ -774,6 +779,10 @@ def get_data(self, data_address: str) -> Any:
         try:
             data_value = reduce(lambda d, key: d[key], element_hierarchy,
                                 self.__pool)
+
+            timestamp = Utility.get_timestamp(include_millis=True)
+            self.__get_data_logs_pool[timestamp] = f"InputManager.get_data() gets called for {element_hierarchy}."
```

Comment:
```suggestion
            self.__get_data_logs_pool[timestamp] = f"InputManager.get_data() called for {element_hierarchy}."
```
I think this makes the message sound a little better.